src/routes/delegates/entities/delete-delegate.entity.ts

```diff
@@ -7,10 +7,4 @@ export class DeleteDelegateDto {
   delegator: string;
   @ApiProperty()
   signature: string;
-
-  constructor(delegate: string, delegator: string, signature: string) {
-    this.delegate = delegate;
-    this.delegator = delegator;
-    this.signature = signature;
-  }
```

Comment:
Not strictly related to this PR, but the coverage report indicated this constructor was never used.